templates/admin-page.php

```diff
@@ -22,6 +22,10 @@
 					</label>
 				</h2>
 
+				<p>
```

Comment:
Maybe we should give this an `id` attribute and then in the `select` provide `aria-describedby` that points to this description? That would be better for accessibility.